### PR TITLE
TexasInstruments.TIConnectCE version 6.0.1.344

### PR DESCRIPTION
**Pull request by tandyx**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140250)

### CODE REVIEW

File: manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE
```
Comment (Trenly) [inline manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: TexasInstruments.TIConnect.CE
```
After updating this, you will need to rename the files and place them in a new folder

File: manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE
```
Comment (Trenly) [inline manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.yaml:4]:
```suggestion
PackageIdentifier: TexasInstruments.TIConnect.CE
```
After updating this, you will need to rename the files and place them in a new folder

File: manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE
+PackageVersion: 6.0.1.344
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{3439CFD2-E613-4729-80BB-6770F3C063A4}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/ti-connect/6.0.1/TIConnectCE-6.0.1.344.msi
```
Comment (Trenly) [inline manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml:11]: This doesn't look like the official TI Link. Please include the PackageUrl in the defaultLocale file

File: manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE
+PackageVersion: 6.0.1.344
```
Comment (Trenly) [inline manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml:5]: The latest version on TI's website is `6.0.0.2688`, is this a beta version?

**Review by Trenly — CHANGES_REQUESTED** [review]
This package does not install unattended, it is blocked by a driver installation prompt

image

**Comment by Trenly** [comment]
[Policy] Interactive Only

File: manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml (hunk 4 above)
Comment (johan-cho) [reply manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml:5]: I got it off here:
https://education.ti.com/en/software/details/en/CA9C74CAD02440A69FDC7189D7E1B6C2/swticonnectcesoftware#!

This is the latest version. I'll look into the driver software install as well, and change the link, may be easier to do in a new pr?

**Comment by johan-cho** [comment]
@Trenly I don't think there's a way to get around the driver install prompt. I didn't see it at first because I had already trusted TI on my machine. I'm going to close this PR.

**Pull request closed without merging**